scipy/stats/_continuous_distns.py

```diff
@@ -5240,10 +5240,28 @@ def _argcheck(self, c):
         return (c >= 0) & (c <= 1)
 
     def _pdf(self, x, c):
-        return np.where(x < c, 2*x/c, 2*(1-x)/(1-c))
+        r = _lazyselect([c == 0,
+                         c == 1,
+                         x < c,
+                         x >= c],
+                        [lambda x, c: 2 - 2 * x,
```

Comment:
style nitpick: the preferred style (and current PEP8 recommendation) is to group things a bit more: `2 - 2*x`, `2 * (1-x) / (1-c)`, etc.